q2_phylogeny/citations.bib

```diff
@@ -10,3 +10,38 @@ @article{price2010fasttree
   publisher={Public Library of Science},
   doi={10.1371/journal.pone.0009490}
 }
+
+@article{Stamatakis2014raxml,
+author = {Stamatakis, Alexandros},
+title = {RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies},
+journal = {Bioinformatics},
+volume = {30},
+number = {9},
+pages = {1312-1313},
+year = {2014},
+doi = {10.1093/bioinformatics/btu033},
+URL = {http://dx.doi.org/10.1093/bioinformatics/btu033},
+}
+
+@article{Nguyen2015iqtree,
```

Comment:
Please see my question above about iqtree.

Reply:
Yarp. I'll remove the iqtree citations for this. :-)